docs/remote_inference_blueprints/standard_blueprints/bedrock_connector_cohere_cohere.embed-english-v3_standard_blueprint.md

```diff
@@ -0,0 +1,190 @@
+# Bedrock connector standard blueprint example for Cohere text embedding model
+This blueprint demonstrates how to deploy a cohere.embed-english-v3 using the Bedrock connector without pre and post processing functions.
+This is recommended for models to use the ML inference processor to handle input/output mapping.
+Note that if using a model that requires pre and post processing functions, you must provide the functions in the blueprint. Please refer to legacy blueprint: [Bedrock connector blueprint example for Cohere embed-english-v3 model](https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/bedrock_connector_cohere_cohere.embed-english-v3_blueprint.md)
+
+## 1. Add connector endpoint to trusted URLs:
+
+Note: no need to do this after 2.11.0
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://bedrock-runtime\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
+        ]
+    }
+}
+```
+
+## 2. Create connector for Amazon Bedrock:
+
+If you are using self-managed Opensearch, you should supply AWS credentials:
```

Comment:
Thinking out loud, rather than doubling the number of blueprints based on AWS vs. self-managed, can we just persist one, and have a single call-out to the documentation describing how it is set up differently between AWS and self-managed? That should greatly reduce the amount of content.

Reply:
ummm we already have AWS doc that talks about how to set up connectors on AWS. But I keep this just to be consist with legacy blueprints

Reply:
Right exactly, was thinking to have just a single blueprint and one callout on the different ways to handle just the credential field, just as a general improvement over the legacy blueprints.